mbuild/packing.py

```diff
@@ -133,8 +166,16 @@ def fill_box(compound, n_compounds=None, box=None, density=None, overlap=0.2,
         compound = [compound]
     if n_compounds is not None and not isinstance(n_compounds, (list, set)):
         n_compounds = [n_compounds]
+
     if not isinstance(fix_orientation, (list, set)):
         fix_orientation = [fix_orientation]*len(compound)
+    try:
+        iter(fix_orientation[0])
+    except:
```

Comment:
```suggestion
    except TypeError:
```
this was in my review from a long time ago but i never hit submit. Anyway...